bin/ch-test

```diff
@@ -900,7 +900,7 @@ if [[ -z $CHTEST_INSTALLED ]]; then
     if [[ $ch_version != "$cf_version" ]]; then
         warning "inconsistent configure version: ${cf_version}"
     fi
-    src_version=$("${ch_base}/misc/version")
+    src_version=$(cat ./VERSION)
```

Comment:
Unclear whether this new comparison will always pass now, making this if statement useless. @reidpr, @heasterday, is checking the VERSION file a suitable alternative for the git usage in `./misc/version`?

Reply:
I'm still familiarizing myself with the original check. As a note though I believe you want `cat ../VERSION` or `cat 
"${ch_base}/VERSION"`as this file lives up a directory from `ch-test` itself.